lambdas/indexer/.chalice/config.json.template.py

```diff
@@ -11,7 +10,9 @@
     "api_gateway_stage": config.deployment_stage,
     "manage_iam_role": False,
     "iam_role_arn": "${var.role_arn}",
-    "environment_variables": aws.lambda_env,
+    "environment_variables":
+        config.lambda_env(es_endpoint=("${var.azul_es_domain.endpoint}", 443),
```

Comment:
To see TF syntax in this file still irks me. I think we should inject those two variables in `prepare_lambda_deployment` and eliminate the arguments to lambda_env